php-frontend/src/main/java/org/sonar/plugins/php/api/tree/declaration/ClassDeclarationTree.java

```diff
@@ -52,6 +52,12 @@
 public interface ClassDeclarationTree extends StatementTree {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move constants to a class or enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1214)